server/workspace_server.go

```diff
@@ -261,3 +261,39 @@ func (s *WorkspaceServer) DeleteWorkspace(ctx context.Context, req *api.DeleteWo
 
 	return &empty.Empty{}, err
 }
+
+// ListNodePools returns a list of the node pools as defined in the system config
+func (s *WorkspaceServer) ListNodePools(ctx context.Context, req *api.ListNodePoolRequest) (*api.ListNodePoolResponse, error) {
+	client := getClient(ctx)
```

Comment:
I think we need an endpoint for config (not namespaced) which will return all system config information that is not a secret. We'll have an equivalent one for namespace configs later too.

Reply:
@rushtehrani would this just be the default namespace configmap?

Reply:
> would this just be the default namespace configmap?

No, this is in `onepanel` namespace.